libs/src/LSPlugin.ts

```diff
@@ -551,6 +554,12 @@ export interface IEditorProxy extends Record<string, any> {
     namespace: BlockPageName
   ) => Promise<Array<PageEntity> | null>
 
+  /**
+   * Create an uuid string for specific block id(uuid)
```

Comment:
```suggestion
   * Create a unique UUID string which can then be assigned to a block.
```
A little unclear. I think this clarifies the language a bit.